source/slang/hlsl.meta.slang

```diff
@@ -4216,6 +4364,10 @@ struct $(item.name)
         }
     }
 
+    /// Load two 32-bit unsigned integers from the buffer at the specified location with alignment
+    /// of `uint2`.
```

Comment:
Speak out what the alignment is, I. This case it is 8.

Reply:
This seems still 8.
I just checked:
source/slang/slang-ir-layout.cpp:236